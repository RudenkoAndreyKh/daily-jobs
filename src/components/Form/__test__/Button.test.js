import React from 'react';
import Button from '../Button';
import renderer from 'react-test-renderer';

it('Button click', () => {
  const component = renderer.create(
    <Button label={'Gag'}>Click</Button>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})