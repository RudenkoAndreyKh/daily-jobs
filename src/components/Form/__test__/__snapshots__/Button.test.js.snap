// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button click 1`] = `
<button
  data-testid="button"
>
  Gag
</button>
`;
